kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/linux/util/LinuxProcessUtil.java

```diff
@@ -351,4 +393,22 @@ public static String getInputStreamAsString(InputStream stream) throws IOExcepti
         }
         return sb.toString();
     }
+    
+    public static boolean waitProcess(int pid, long poll, long timeout) {
```

Comment:
@ibinshtok What is the difference between this waitProcess and the one in PppLinux?